src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -2274,6 +2274,7 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
 
   public loadPerson(person: Person, id: string, jwt: string) {
     this.donor = person; // Should mean donations are attached to the Stripe Customer.
+    this.paymentStepErrors = "";
```

Comment:
this is not necessary along with the change in the container, but I think is worth keeping just in case.